apps/prairielearn/src/pages/administratorFeatures/administratorFeatures.html.ts

```diff
@@ -47,7 +47,7 @@ export function AdministratorFeatures({
         })}
         <main id="content" class="container">
           <div class="card mb-4">
-            <div class="card-header bg-primary text-white">Features</div>
+            <h1 class="card-header bg-primary text-white h6 font-weight-normal">Features</h1>
```

Comment:
For card headers without additional buttons, I updated the `<div>` to be the `<h1>` heading as to reduce the amount of elements used here. Let me know if you would prefer the `<h1>` to be a child of the `<div>` for these instead.

Reply:
I would indeed prefer to nest `<h1>` inside the `div.card-header`. This will make sure that we don't accidentally end up announcing action buttons as part of the heading if we add them later.

Reply:
Sorry, I should have done this on the first go around. Updated to nesting the `<h1>` in [2d90727](https://github.com/PrairieLearn/PrairieLearn/pull/10440/commits/2d907275890a1dae887227f7e0d2c248110fa04e).